app/pages/covid19/data/non-production/covid19-solutions.json

```diff
@@ -1,73 +1,309 @@
+        ]
+      }
+    },
+    {
+      "id": "10072-006",
+      "name": "Push Access",
+      "summary": "Push Access allows a patient to request and receive support relating to their healthcare concerns online. Allowing the Healthcare professional dealing with the request to triage requests effectively and invite the patient for a phone, video or face-2-face consultation if appropriate.",
+      "supplier": {
+        "id": "10072",
+        "name": "Push Dr Ltd"
+      },
+      "covid19": {
+        "title": "How this Catalogue Solution can help with coronavirus",
+        "list": [
+          "This Catalogue Solution has been assessed for video consultation",
+          "Lets patients request support online",
+          "Allows appointment triage based on what a patient needs from a consultation",
+          "This Catalogue Solution is not covered by a central contract and would therefore need to be purchased directly in the usual way",
+          "This Catalogue Solution has met an eConsultation Capability"
+        ]
+      }
+    },
+    {
+      "id": "10072-004",
+      "name": "Digital Locum",
+      "summary": "Patients are able to view and book appointments via their app. Practices are able to schedule appointments to appropriate patients. Includes automated tech checks, ID checks and pre-appointment reminders and a virtual waiting room.",
+      "supplier": {
+        "id": "10072",
+        "name": "Push Dr Ltd"
+      },
+      "covid19": {
+        "title": "How this Catalogue Solution can help with coronavirus",
+        "list": [
+          "This Catalogue Solution has been assessed for video consultation",
+          "Allows patients to view and book appointments online",
+          "Secure patient ID verification",
+          "This Catalogue Solution is not covered by a central contract and would therefore need to be purchased directly in the usual way",
+          "This Catalogue Solution has met a Productivity Capability"
+        ]
+      }
+    },
+    {
+      "id": "10063-002",
+      "name": "forms4health",
+      "summary": "forms4health™ is an intuitive, easily integrated, electronic smart forms platform, facilitating paperless working across health and social care. forms4health is a responsive, versatile and interoperable platform that delivers digital transformation.",
+      "supplier": {
+        "id": "10063",
+        "name": "Aire Logic"
+      },
+      "covid19": {
+        "title": "How this Catalogue Solution can help with coronavirus",
+        "list": [
+          "Allows the design of bespoke eConsultations",
+          "Quick and easy implementation",
+          "This Catalogue Solution is not covered by a central contract and would therefore need to be purchased directly in the usual way",
+          "This Catalogue Solution has met a Productivity Capability and partially met two eConsultation Capabilities"
         ]
       }
     }
   ]
-}
+}
```

Comment:
Would you mind adding a new line to make Github happy? Does linting not cover json files?